domains/_discord.painscreen.json

```
@@ -1,8 +1,9 @@
 {
         "owner": {
            "username": "iHategithub9000",
-           "email": "stefeksiekiera@gmail.com",
-           "discord": "836012649959522384"
+           "email": "",
+           "discord": "id: 836012649959522384, username: yourhardworkgone, displayname: painscreen, displayname on server: PainScreen5695",
```

You just need the discord ID mate
```suggestion
           "discord": "836012649959522384",
```